### ISSUE TITLE
Errors out when trying to download

### ISSUE THREAD

**Issue by nf33** [oleksis/youtube-dl-gui#2]
The download "completes" and errors out at 0 percent when trying to dl from youtube.  Latest version.  Windows 7

**Comment by h3xpl01t** [comment]
Yeah, its already reported [here](https://github.com/oleksis/youtube-dl-gui/issues/1). 

I'm going to be assuming what's causing the problem here, as I am in no way connected to the devs. But I'm also pretty sure I know whats wrong, considering that the cause of the issue happened in the past month or so and the last update was about 3 months ago. 

Youtube-dl-gui is attempting to download the latest binaries from the GitHub Repo which is currently down. The latest updates for the youtube-dl binaries are currently being hosted on their website, [here for Windows](https://youtube-dl.org/downloads/latest/youtube-dl.exe) and [here for Linux](https://youtube-dl.org/downloads/latest/youtube-dl). The source is available [here](https://youtube-dl.org/downloads/latest/youtube-dl-2020.11.12.tar.gz) as well. 

As a temporary fix for the situation so you can still use the GUI as normal, you can download the binaries manually and replace the current ones, while also remembering to disable automatic updates until it gets fixed.



The short version of why this is happening is because youtube-dl got DMCA'd off GitHub by everyone's favorite destroyer of fun and happiness, the RIAA. Its absolutely ridiculous, but a complaint was made that youtube-dl could be used to download copyrighted music. As if any sane human would ever want to have a whole collection of garbage-quality lossy-to-lossy transcoded music. 

If you're interested and wanna read up on this absolutely ludicrous situation, you can check out [this article](https://www.eff.org/deeplinks/2020/11/github-youtube-dl-takedown-isnt-just-problem-american-law) on this situation by the EFF, or [this article](https://hackaday.com/2020/10/27/community-rallies-behind-youtube-dl-after-dmca-takedown/) by Hackaday.

**Comment by h3xpl01t** [comment]
@nf33 You can actually disregard my last comment, everything is finally sorted out. YouTube-DL GUI should now function as expected when trying to update

**Comment by oleksis** [comment]
> The download "completes" and errors out at 0 percent when trying to dl from youtube. Latest version. Windows 7

See like @h3xpl01t say https://github.com/oleksis/youtube-dl-gui/issues/2#issuecomment-727641748 fix your problem for close the issue.
